gatorgrade/main.py

```diff
@@ -16,22 +20,23 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
-        if filename.suffix == "yml":
-            pass
+        checks = parse_config(FILE)
```

Comment:
You should use the `filename` parameter here, instead of the hard-coded constant. Additionally, it looks like the default value for `filename` was reverted back to an older value -- it should be the value of `FILE`, not the string "FILE".